packages/appstore/package.json

```diff
@@ -31,6 +31,7 @@
         "@deriv/trader": "^3.8.0",
         "@deriv/translations": "^1.0.0",
         "@deriv/ui": "^0.0.1",
+        "@radix-ui/react-dialog": "^0.1.5",
```

Comment:
@vinu-deriv instead of adding `@radix-ui/react-dialog` to our dependency, can we try to export `Dialog` from `@deriv/ui` and consume it here?